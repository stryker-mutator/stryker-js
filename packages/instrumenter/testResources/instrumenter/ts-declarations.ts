export declare const globalNamespace = "globalNamespace";
declare function foo(): 'foo';
declare module 'express' {
}

React.useState<false>();
