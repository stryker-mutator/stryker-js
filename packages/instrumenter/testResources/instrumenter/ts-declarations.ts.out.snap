// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`instrumenter integration type declarations should not produce mutants for a TS declaration file 1`] = `
"export declare const globalNamespace = \\"globalNamespace\\";
declare function foo(): 'foo';
declare module 'express' {}
React.useState<false>();"
`;
